src/ec/packages/ec-component-text-input/text-input.html.twig

```diff
@@ -60,30 +66,38 @@
 <div class="ecl-form-group ecl-form-group--text-input">
   {% if _label is not empty %}
     <label
-      class="ecl-form-label{{ invalid ? ' ecl-form-label--invalid' }}{{ hide_label ? ' ecl-form-label--hidden' }}"
+      class="ecl-form-label
+      {{- _invalid ? ' ecl-form-label--invalid' -}}
+      {{- _disabled ? ' ecl-form-label--disabled' -}}
+      "
       for="{{ _id }}"
     >
       {%- block label _label -%}
+      {%- if _required %}
+        <span class="ecl-form-label__required">{{ _required_text }}</span>
```

Comment:
aaaargghhhh, we are struggling a lot with this logic:
we render the optional text span only when the field is not mandatory and when the optional text is not empty and we render the mandatory one only when the field is required and the required text is not empty

Reply:
ok now it should be good :)